tests/README.md

```diff
@@ -0,0 +1,31 @@
+# Streisand-CI testing
+Streisand CI testing. Uses travis-ci to test Ansible syntax and kick off a full playbook run.
+Full playbook run uses LXC to execute.
+
+## Limitations
+The TOR playbook role is skipped. Turns out the travis-ci environment doesn't like being added to the tor network (even temporarily for testing purposes).
+Streisand doc generation is also skipped for now until some mock variables are added for the skipped TOR role.
+
+## Local Testing
+You can use this testing framework locally as well (Locally, the TOR playbook will be ran).
+
+Environment Expectations:
```

Comment:
When you say Environment here you mean the host environment that will be running Ansible? Can you clarify that since experience shows people are easily confused about which machine you're talking about when you're dealing with Ansible, containers, and Streisand servers.